package.json

```diff
@@ -94,8 +97,9 @@
   "scripts": {
     "postinstall": "./tools/task-runner/runner githooks --verbose",
     "sass-watch": "node-sass -w ./static/src/stylesheets -o ./static/target/stylesheets --source-map=true",
-    "css-watch": "gulp --cwd ./dev watch:css",
+    "babel-watch": "BABEL_ENV=production babel ./static/src/javascripts --watch -d ./static/transpiled/javascripts --ignore .eslintrc.js >/dev/null 2>&1",
```

Comment:
Why do we use a `BABEL_ENV` of `production` for the watch task? I guess because it's the same as what happens on production, but there's also a `development` one which I'm not sure is used?